rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -0,0 +1,80 @@
+/* Component which renders the x,y,z axes along with the grid lines along the xy plane. Used within
+Data_Association_PC.js.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Third-Party Package Imports.
+import {Line, Html} from "drei";  // Used to render axes lines and text.
+
+const lowerGridBound = -10;   // Lower grid bound for coordinate axes.
```

Comment:
Consider making these file-level constants default arguments to the function below.